packages/platform/platform-log-middleware/readme.md

```
@@ -20,7 +20,7 @@
   <span>&nbsp;&nbsp;•&nbsp;&nbsp;</span>
   <a href="https://tsed.dev/getting-started/">Getting started</a>
   <span>&nbsp;&nbsp;•&nbsp;&nbsp;</span>
-  <a href="https://slack.tsed.io">Slack</a>
+  <a href="https://slack.tsed.dev">Slack</a>
```

markdownlint-cli2 (0.17.2)

Inline HTML
Element: a

(MD033, no-inline-html)